tutorcelery/patches/k8s-deployments

```diff
@@ -0,0 +1,55 @@
+{% for service, variants in CELERY_WORKER_VARIANTS.items() %}
+
+{% for variant in variants%}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{service}}-worker-{{variant}}
+  labels:
+    app.kubernetes.io/name: {{service}}-worker-{{variant}}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{service}}-worker-{{variant}}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{service}}-worker-{{variant}}
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+      containers:
+        - name: {{service}}-worker-{{variant}}
+          image: {{ DOCKER_IMAGE_OPENEDX }}
+          args:
+            - "celery"
+            - "--app={{service}}.celery"
+            - "worker"
+            - "--loglevel=info"
+            - "--hostname=edx.{{service}}.core.{{variant}}"
```

Comment:
```suggestion
            - "--hostname=edx.{{service}}.core.{{variant}}.%%h"
```
Should it be like the [Tutor hostname](https://github.com/overhangio/tutor/blob/master/tutor/templates/k8s/deployments.yml#L144C110-L144C113)?